pkg/collection/ducks_test.go

```diff
@@ -199,6 +239,60 @@ func Test_DuckHunter_AddCRD_filtered(t *testing.T) {
 				}},
 			},
 		},
+		"match second label": {
+			dh: NewDuckHunter(nil, nil, &DuckFilters{
+				DuckLabel:         "teach.me.how/ducky",
+				DuckVersionPrefix: "duckies.teach.me.how",
+			}),
+			crd: makeCRDAnnotated("teach.me.how", "Ducky", map[string]bool{"v2": true},
+				map[string]string{"you.know.how": "whatever", "teach.me.how/ducky": "true"}, map[string]string{"duckies.teach.me.how/v1": "v2"}),
+			want: map[string][]v1alpha1.ResourceMeta{
+				"v1": {{
+					APIVersion: "teach.me.how/v2",
+					Kind:       "Ducky",
+					Scope:      "Namespaced",
+				}},
+			},
+		},
+		"reject match filter": {
```

Comment:
Without more context I find it difficult to understand what's being tested here. Care to use test values that better highlight the important bits?

Reply:
the label is set to a false: `"teach.me.how/ducky": "false"`

Reply:
I guess what I was expecting was a label called `this.will.be/rejected` for example, but it's also me being unfamiliar with the current impl so I'm not feeling strongly about it.